---
published: true
layout: secondary
title: Data
section_id: data

data:
  publication:
    title: Spatial intra-tumor heterogeneity is associated with survival of lung adenocarcinoma patients
    authors: 'Wu HJ, Temko D, Maliga M, Moreira A, Sei E, Conterno Minussi D, Dean J, Lee C, Xu Q, Hochart G, Jacobson C, Yapp C, Schapiro D, Sorger P, Seeley EH, Navin N, Downey RJ, and Michor F'
    journal: 'Cell Genomics. 2022 Aug; 2(8): 100165. PMID: 35404441'
    description: Intratumor   heterogeneity   (ITH)   of   human   tumors   is   important   for   tumor progression,   treatment   response,   and   drug   resistance.   However,   the   spatial distribution of ITH remains incompletely understood. Here, we present spatial analysis of ITH in lung adenocarcinomas from 147 patients using multi-region mass spectrometry of >5000 regions, single cell copy number sequencing of ~2000 single cells, and cyclic immunofluorescence of >10 million cells. We identified two distinct spatial   patterns   among   tumors,   termed   clustered   and   random   geographic diversification (GD). These patterns were observed in the same samples using both proteomic and genomic data. The random proteomic GD pattern, which is characterized by decreased cell adhesion and lower levels of tumor-interacting endothelial cells, was significantly associated with increased risk of recurrence or death in two independent patient cohorts. Our study presents comprehensive spatial mapping of ITH in lung adenocarcinoma and provides insights into the mechanisms and clinical consequences of geographic diversification of intratumor heterogeneity.
    links:
      - Publication: https://doi.org/10.1016/j.xgen.2022.100165
---
{% assign urlParts = page.url | split: '/' %}
{% assign sectionId = urlParts[-1] %}

{% include atlas-dataset-info.html
    sectionId=sectionId
    pubData=page.data
    thumbnailDir=sectionId %}

### Contents
* [Data Overviews](#data-overviews)
* [Data Access](#data-access)

## Data overviews

**Data Overviews provide access to minimally processed Level 2 images with no annotation or quality control. Click any of the following thumbnail images for an interactive view of the full-resolution images.**

{%
    assign stories = site.data-cards
    | where_exp: "item", "item.url contains 'wu-temko-maliga-2022/'"
    | where_exp: "item", "item.hide != true"
%}

{% assign dataCardArray = '' | split: '' %}
{% for s in stories %}
  {% unless s.url contains '-overview' %}
    {% assign dataCardArray = dataCardArray | push: s %}
  {% endunless %}
{% endfor %}

{% if dataCardArray.size > 0 %}
  {% include cards.html cards=dataCardArray %}
{% endif %}

## Data Access
### About the Data Files

This dataset can be used to reproduce the analysis of 12 lung tumors, as described (Wu HJ, Temko D, Maliga Z, et. al. Cell Genomics, 2022), and to develop new computational tools for processing and analysis of tumors stained for standard histopathology (hematoxylin and eosin) or highly multiplexed (20+) fluorescent tissue imaging.

The dataset includes raw brightfield and immunofluorescence images of surgical resections of lung adenocarcinoma from twelve patients. The stitched image pyramids for highly multiplexed fluorescence image data, segmentation masks and single-cell intensity files for each sample generated by MCMICRO is included.

The position of histologically defined regions (20-60 per tumor) in the H&E stained sections and the corresponding region in the adjacent section stained by multiplexed fluorescence microscopy (CyCIF) are indicated.

<details>
    <summary>Download the primary data</summary>
<div markdown="1">
{% include_relative wu-temko-maliga-2022-file-list.md %}
</div>
</details>
